infra/aws/terraform/prow-build-cluster/eks.tf

```diff
@@ -30,6 +30,17 @@ module "eks" {
   # Manage aws-auth ConfigMap.
   manage_aws_auth_configmap = true
 
+  # Configure aws-auth
+  aws_auth_roles = [
+    {
+      "rolearn"  = aws_iam_role.eks_admin.id
+      "username" = "eks-admin"
```

Comment:
Where this value comes from? Should that eventually be name of IAM role or that's something completely else? I'm still trying to understand how this ConfigMap works at all.

Reply:
This is an arbitary string. https://github.com/kubernetes-sigs/aws-iam-authenticator#crd-alpha